website/content/docs/release-notes/consul-k8s/v0_48_x.mdx

```diff
@@ -0,0 +1,61 @@
+---
+layout: docs
+page_title: 0.47.x
+description: >-
+  Consul on Kubernetes release notes for version 0.48.x
+---
+
+# Consul on Kubernetes 0.48.0
+
+## Release Highlights
+
+- **Consul CNI Plugin**: This release introduces the Consul CNI Plugin for Consul on Kubernetes, to allow for configuring traffic redirection rules without escalated container privileges such as `CAP_NET_ADMIN`. Refer to [Enable the Consul CNI Plugin](docs/k8s/installation/install#enable-the-consul-cni-plugin) for more details. The Consul CNI Plugin is supported for Consul K8s 0.48.0+ and Consul 1.13.1+.
```

Comment:
I don't think the `docs` link will work if it doesn't start with a `/`.

Reply:
Thank you!